operator/src/main/java/org/keycloak/operator/v2alpha1/KeycloakController.java

```diff
@@ -94,6 +111,12 @@
         kcDiscoveryService.updateStatus(statusBuilder);
         kcDiscoveryService.createOrUpdateReconciled();
 
+        if (!kc.getSpec().isDefaultIngressDisabled()) {
+            var kcIngress = new KeycloakIngress(client, kc);
+            kcIngress.updateStatus(statusBuilder);
+            kcIngress.createOrUpdateReconciled();
+        }
```

Comment:
We should delete if `kc.getSpec().isDefaultIngressDisabled() == true`.

Reply:
I would play it safe and let the deletion of the ingress to be triggered only by `OwnerReference`s, or as a manual action from the user.
wdyt?

Reply:
The operators should be idempotent. If we won't delete it, we're breaking this rule because the result of `isDefaultIngressDisabled = false` depends on whether the Ingress object already exists or not.

Reply:
I'm still not sure this is the right behaviour, still, I don't have arguments strong enough to oppose, let's go down this way and re-iterate!
(Only note is that this operator is really not idempotent)